psse/psse-converter/src/main/java/com/powsybl/psse/converter/GeneratorConverter.java

```diff
@@ -100,8 +103,8 @@ private static Terminal defineRegulatingTerminal(PsseGenerator psseGenerator, Ne
             }
         }
         if (regulatingTerminal == null) {
-            throw new PsseException("PSSE. Generator " + defaultRegulatingBusId + "-"
-                + getGeneratorId(defaultRegulatingBusId, psseGenerator) + ". Unexpected regulatingTerminal.");
+            String generatorId = getGeneratorId(defaultRegulatingBusId, psseGenerator);
+            LOGGER.warn("PSSE. Generator {}. Regulating terminal is not assigned as the bus is isolated", generatorId);
```

Comment:
I am not sure that you need to mention `PSSE`, the LOGGER makes it for you.

Reply:
Done